packages/shared/providers.yaml

```diff
@@ -4893,6 +4893,65 @@ sage:
         base_url: https://api.accounting.sage.com
     docs: https://docs.nango.dev/integrations/all/sage
 
+sage-intacct:
+    display_name: Sage Intacct
+    categories:
+        - accounting
+        - erp
+    auth_mode: TWO_STEP
+    proxy:
+        base_url: https://api.intacct.com/ia/xml/xmlgw.phtml
+    token_url: https://api.intacct.com/ia/xml/xmlgw.phtml
+    body_format: xml
+    token_params:
+        request:
+            control:
+                senderid: ${credential.senderId}
+                password: ${credential.senderPassword}
+                controlid: ${timestamp}
```

Comment:
is it really a timestamp? do you have associated doc?

Reply:
They recommend using a `timestamp` for this value as it is a good way to differentiate similar [requests](https://developer.intacct.com/web-services/requests/#controlid-element-required). 
https://developer.intacct.com/web-services/your-first-api-calls/

Reply:
what about ${timestamp.now} or `${current_timestamp}` or just `${now}` to be clear about what the timestamp value  is going to be